Added Document for VMware Tanzu Application Service (#3775)
SumoLogic/sumologic-documentation: docs/integrations/containers-orchestration/vmware-tanzu-application-service.md

---
id: vmware-tanzu-application-service
title: VMware Tanzu Application Service
sidebar_label: VMware Tanzu Application Service
description: This documentation describes the Sumo Logic Nozzle for VMware Tanzu tile.
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('img/integrations/containers-orchestration/vmware-tanzu.png')} alt="VMware Tanzu Application Service" width="50" />

The Sumo Logic Nozzle for VMware Tanzu can help you:

* View dashboards to monitor the overall health and security of your VMware Tanzu resources
* Drill down from alerts to see key performance metrics and audit logs for VMware Tanzu and your custom apps
* Store VMware Tanzu audit details in a fully secure SOC 2, HIPAA, and PCI-compliant environment

> The Sumo Logic nozzle for [VMware Tanzu Firehose](https://docs.vmware.com/en/Tile-Developer-Guide/3.0/tile-dev-guide/nozzle.html) allows operations teams and app owners to monitor and troubleshoot the VMware Tanzu platform and apps in real time, without having to install a Sumo Logic agent in their Cloud Foundry environment.

Reviewer: ```suggestion
The Sumo Logic Nozzle for [VMware Tanzu Firehose](https://docs.vmware.com/en/Tile-Developer-Guide/3.0/tile-dev-guide/nozzle.html) allows operations teams and app owners to monitor and troubleshoot the VMware Tanzu platform and apps in real time, without having to install a Sumo Logic agent in their Cloud Foundry environment.
```

---
id: vmware-tanzu-application-service
title: VMware Tanzu Application Service
sidebar_label: VMware Tanzu Application Service
description: This documentation describes the Sumo Logic Nozzle for VMware Tanzu tile.
---
import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('img/integrations/containers-orchestration/vmware-tanzu.png')} alt="VMware Tanzu Application Service" width="50" />

> ## <a id='nozzle-overview'></a> Overview of Sumo Logic Nozzle for VMware Tanzu
> 
> Sumo Logic Nozzle for VMware Tanzu can help you:

Reviewer: @akhil-sumologic , our docs style is to omit the "overview" header and go straight into the overview description

```suggestion
The Sumo Logic Nozzle for VMware Tanzu can help you:
```

Author: > @akhil-sumologic , our docs style is to omit the "overview" header and go straight into the overview description

Addressed
